tests/Feature/Controllers/Api/Journals/Modules/WeatherInfluence/WeatherInfluenceControllerTest.php

```
@@ -0,0 +1,178 @@
+        $this->assertEquals('significant', $entry->moduleWeatherInfluence->plans_influence);
+        $this->assertEquals('barely', $entry->moduleWeatherInfluence->outside_time);
+    }
+
+    #[Test]
+    public function it_validates_mood_effect_must_be_valid(): void
+    {
+        $user = User::factory()->create();
+        $journal = Journal::factory()->create([
+            'user_id' => $user->id,
+        ]);
+        $entry = JournalEntry::factory()->create([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$entry'. (undefined)

(UnusedLocalVariable)